packages/typescript-configs/core/react.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "extends": "./base.json",
+    "compilerOptions": {
+        "moduleResolution": "bundler",
+        "jsx": "react-jsx",
+        "lib": ["dom", "dom.iterable", "esnext"],
+        "module": "ESNext",
```

Comment:
I would expect everything related to ESM to be in the "base" config at it's our default.